prymer/offtarget/bwa.py

```diff
@@ -297,10 +306,9 @@ def __init__(
 
     def __signal_bwa(self) -> None:
         """Signals BWA to process the queries"""
-        for _ in range(3):
-            self._subprocess.stdin.flush()
-            self._subprocess.stdin.write("\n\n")
-            self._subprocess.stdin.flush()
+        self._subprocess.stdin.flush()
+        self._subprocess.stdin.write("\n" * 16)
+        self._subprocess.stdin.flush()
```

Comment:
The Bioconda built executable requires more newlines before emitting an alignment.

Why the executable (regardless of compilation environment) needs more than 2 newlines is currently a mystery.